JapaneseFilter/sections/specific.txt

```diff
@@ -1154,6 +1154,9 @@ natuyuki.com##.cta
 i486.mods.jp##table[style$="width: 300px;"]
 rocomotion.net###maintitle_ad
 retty.me##.dfp-banners
+retty.me##div[style^="margin: 30px auto 0;"]
+retty.me##.middle-ad-section
+retty.me##.small-ad-content
```

Comment:
What about change that two rules to  `##div[class*="-ad"]` rule ? You can add it for: `kai-you.net##div[class*="-ad"]`

Reply:
I see now.

Reply:
@gork7777  thank you for good advice

Reply:
@gork7777 So are you the one that decides how to fix reports? There is nothing here that concerns you. And stop sticking your nose into pull requests.
Developers can handle this